api/v1alpha1/etcdpeer_types.go

```diff
@@ -32,7 +32,7 @@ type Bootstrap struct {
 	// Static boostrapping requires that we know the network names of the
 	// other peers ahead of time.
 	// +optional
-	Static *StaticBootstrap `json:"static,omitempty"`
+	Static StaticBootstrap `json:"static,omitempty"`
```

Comment:
This should remain a pointer - there may be additional bootstrap mechanisms supported in future and so there is a `oneOf` relationship on the fields in `Bootstrap`.

Reply:
Yeah, but right now we don't support other bootstrap mechanisms so it's required.
And as it stands there will be a panic at https://github.com/improbable-eng/etcd-cluster-operator/pull/68/files#diff-8de33edb64246a5a97f474b86fefda9d if this is allowed to be nil.